tests/service/data/SyntaxTree/UnionCase/Missing name 06.fs.bsl

```diff
@@ -0,0 +1,38 @@
+ImplFile
+  (ParsedImplFileInput
+     ("/root/UnionCase/Missing name 06.fs", false, QualifiedNameOfFile Module,
+      [], [],
+      [SynModuleOrNamespace
+         ([Module], false, NamedModule,
+          [Types
+             ([SynTypeDefn
+                 (SynComponentInfo
+                    ([], None, [], [U],
+                     PreXmlDoc ((3,0), FSharp.Compiler.Xml.XmlDocCollector),
+                     false, None, (3,5--3,6)),
+                  Simple
+                    (Union
+                       (None,
+                        [SynUnionCase
+                           ([], SynIdent (A, None), Fields [],
+                            PreXmlDoc ((4,4), FSharp.Compiler.Xml.XmlDocCollector),
+                            None, (4,6--4,7), { BarRange = Some (4,4--4,5) });
+                         SynUnionCase
+                           ([], SynIdent (, None), Fields [],
+                            PreXmlDoc ((5,4), FSharp.Compiler.Xml.XmlDocCollector),
+                            None, (5,4--5,5), { BarRange = Some (5,4--5,5) });
+                         SynUnionCase
+                           ([], SynIdent (C, None), Fields [],
+                            PreXmlDoc ((6,4), FSharp.Compiler.Xml.XmlDocCollector),
+                            None, (6,6--6,7), { BarRange = Some (6,4--6,5) })],
+                        (4,4--6,7)), (4,4--6,7)), [], None, (3,5--6,7),
+                  { LeadingKeyword = Type (3,0--3,4)
+                    EqualsRange = Some (3,7--3,8)
+                    WithKeyword = None })], (3,0--6,7))],
+          PreXmlDoc ((1,0), FSharp.Compiler.Xml.XmlDocCollector), [], None,
+          (1,0--6,7), { LeadingKeyword = Module (1,0--1,6) })], (true, true),
+      { ConditionalDirectives = []
+        CodeComments = [] }, set []))
+
+(6,4)-(6,5) parse error Unexpected symbol '|' in union case
+(5,6)-(5,14) parse error Accessibility modifiers are not permitted on union cases. Use 'type U = internal ...' or 'type U = private ...' to give an accessibility to the whole representation.
```

Comment:
@auduchinok 

is
````
> type U =
>     | A
-     | internal of
-     | C

      | C
  ----^

stdin(24,5): error FS0010: Unexpected symbol '|' in union case

-   38 ;;
>
````
a better error message than:
````
> type U =
-      | A
-      | internal of

       | internal of
  ----------------^^

stdin(3,17): error FS0010: Unexpected keyword 'of' in union case. Expected identifier, '(', '(*)' or other token.

-      | C
-
````
I like the 
````parse error Accessibility modifiers are not permitted on union cases. Use 'type U = internal ...' or 'type U = private ...' to give an accessibility to the whole representation.```` 
which seems like the best message

Reply:
@KevinRansom Nice catch, thanks. I was expecting `parsMissingUnionCaseName` to be raised at `of` range here, but it didn't work for some reason. I'll look into it.

Reply:
There's now an additional error about missing name.